flexget/plugins/input/next_trakt_episodes.py

```diff
@@ -75,6 +75,7 @@ def on_task_input(self, task, config):
                     trakt_id = item['show']['ids']['trakt']
                     listed_series[trakt_id] = {
                         'series_name': '%s (%s)' % (item['show']['title'], item['show']['year']),
+                        'series_year': item['show']['year'],
```

Comment:
I mean `movie_year` is also handled this way, so why not `series_` why not fill in all valid information we can get at a particular stage?